torchmd/parameters.py

```diff
@@ -164,7 +169,7 @@ def make_lj(self, ff, uqatomtypes):
         A = torch.tensor(A)
         B = torch.tensor(B)
         return A, B
-
+    
```

Comment:
Also install `black` for autoformatting when saving in VS Code, otherwise you end up with stuff like these empty spaces here :)

Reply:
Thanks for the advice!